rel-eng/openshift-sdn-node.sysconfig

```diff
@@ -55,4 +55,4 @@ OPTIONS=
 # The $DOCKER_OPTIONS variable is used to overwrite the $OPTIONS
 # variable in the /etc/sysconfig/docker file. This is a pretty ugly
 # hack, but whatevs
```

Comment:
Should be fixed now, also linked to more elegant solution for anyone willing to invest more time.